### ISSUE TITLE
Wrong sequence used in case of inherited default value

### ISSUE THREAD

**Issue by bordig-f** [orientechnologies/orientdb#7876]
## Expected behavior
After running the following script:
```
create sequence parent_seq type ordered start 1023 increment 1
create sequence child_seq type ordered start 0 increment 1

create class parent extends v
create property parent.parent_id string (notnull true, mandatory true)
alter property parent.parent_id default 'sequence("parent_seq").next()'

create class child extends parent
create property child.child_id string

create vertex child set child_id=sequence("child_seq").next()
select parent_id, child_id from child
```

The two sequences should have been be used:
```
+----+---------+--------+
|#   |parent_id|child_id|
+----+---------+--------+
|0   |1024     |1       |
+----+---------+--------+
```

## Actual behavior
...but the output shows that only parent_seq has been used, twice:
```
+----+---------+--------+
|#   |parent_id|child_id|
+----+---------+--------+
|0   |1024     |1025    |
+----+---------+--------+
```

## Steps to reproduce
Run the script specified above.

**Comment by luigidellaquila** [comment]
Hi @bordig-f 

Thank you for reporting, I'll check it asap

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @bordig-f 

I just pushed a fix to 2.2.x branch, it will be released with 2.2.31

Thanks

Luigi

**Comment by bordig-f** [comment]
Thank you Luigi =)
